assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -606,7 +606,9 @@ che.server.secure_exposer.jwtproxy.token.issuer=wsmaster
 che.server.secure_exposer.jwtproxy.token.ttl=8800h
 che.server.secure_exposer.jwtproxy.auth.loader.path=/_app/loader.html
 che.server.secure_exposer.jwtproxy.image=quay.io/eclipse/che-jwtproxy:0.10.0
+che.server.secure_exposer.jwtproxy.memory_request=10mb
 che.server.secure_exposer.jwtproxy.memory_limit=128mb
+che.server.secure_exposer.jwtproxy.cpu_request=0.02
```

Comment:
Also, might be good to increase the request to `0.03`

![image](https://user-images.githubusercontent.com/1461122/101615883-f4720b80-3a0e-11eb-9e39-8bcbdba27833.png)

Reply:
updated